evals/benchmark/README.md

```diff
@@ -12,12 +12,171 @@ This Tool provides a microservice benchmarking framework that uses YAML configur
 
 ## Table of Contents
 
+- [Workload Deployment in Kubernetes](#workload-deployment-in-kubernetes)
```

Comment:
I was under the impression we keep "deployment" under GenAIExamples, because it is essential to have a single docker node or a kubernetes cluster of one or more nodes to run your GenAIExamples, or to benchmark. ..so move deployment to GenAIExamples.  Which also means you need to cover single docker node - bare metal or VM and give them link to learn how to download docker
Something along the below lines


sudo apt update -y
curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
DEBIAN_FRONTEND=noninteractive sudo -E add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" -y
sudo apt update -y
DEBIAN_FRONTEND=noninteractive sudo -E apt install docker-ce docker-ce-cli containerd.io -y
sudo usermod -aG docker $USER

Reply:
Yes, once we achieve alignment on the out-of-the-box (OOB) flows for benchmarking in Kubernetes, we can proceed with reorganizing other sections. The 1/2/4/8 manifests pertain specifically to the current Kubernetes deployment.